### DISCUSSION TITLE
several s32cinemagoer imports?

### DISCUSSION THREAD

**Discussion by bobblkabb** [cinemagoer/cinemagoer#399] [Q&A] [answered]
Hello,

is it possible to import the tsv.gz into a database not all at once, but as needed?
The imports take a very long time, which is why I would like to split the complete database.

Using postgresql btw. [edit]

**Comment by alberanid** [comment, marked as answer]
Yes, you can import the files separately. Since the script takes a directory as a parameter, you have to put each file in a different directory.
